lib/jurnal_api/client/journal_entries.rb

```diff
@@ -0,0 +1,33 @@
+module JurnalApi
+  class Client
+    # Defines methods related to SalesOrders
+    module JournalEntries
+
+      def journal_entries(params = {})
```

Comment:
How about change naming for methods like this one.

```
def all
end

def find
end

def create
end

def update(id, params)
end

def delete(id)
```

Reply:
agree. Prepending all the `JournalEntries` methods with `journal_entries` is redundant